### ISSUE TITLE
Add info about provider option in --help command

### ISSUE THREAD

**Issue by tropicar** [dappnode/DAppNodeSDK#183]
**Is your feature request related to a problem? Please describe.**

It's a functionality that is very useful and there is no information related to this.

**Describe the solution you'd like**

It would be nice to add this option in the --help command and maybe explain it in the readme it would not be a bad idea.

**Comment by FAvE01** [comment]
👮‍♀️

**Comment by alexpeterson91** [comment]
Agreed. Had no idea how to build a package while ssh'd into my dappnode to install on that node. The docs for the local build option isn't in the sdk help yet. Also regarding the build local flag `--provider http://172.33.1.5:5001` should probably simplify by making an alias for this flag that's shorter and simpler like `--local`

**Comment by pablomendezroyo** [comment]
Take into account that now providers for eth and ipfs are requested
